flexmeasures/cli/monitor.py

```diff
@@ -250,7 +250,9 @@ def monitor_last_seen(
 
     # find users we haven't seen in the given time window
     users: list[User] = db.session.scalars(
-        select(User).filter(User.last_seen_at < datetime.utcnow() - last_seen_delta)
+        select(User).filter(
+            User.last_seen_at < datetime.now(timezone.utc) - last_seen_delta
```

Comment:
Is this line covered by a test? I commented on this line before in PR #541, as it's comparing a naive datetime with an aware datetime. It worked out then, but let's make sure it still works now.

Reply:
Looks like [there is no test covering it](https://coveralls.io/builds/70954451/source?filename=flexmeasures%2Fcli%2Fmonitor.py#L253).

We change here to a non-naive, before they were both naive right?

Reply:
That is my hunch, yes. Is a quick manual test possible?

Reply:
Yes, I tested manually now (while busy with #1268) and it worked.